book/src/configuration.md

```diff
@@ -75,6 +75,7 @@ available, which is not defined by default.
 |`git-global` | Enables reading global .gitignore, whose path is specified in git's config: `core.excludefile` option. | true
 |`git-exclude` | Enables reading `.git/info/exclude` files. | true
 |`max-depth` | Set with an integer value for maximum depth to recurse. | Defaults to `None`.
+|`root` | Set the option decide how to find root dir, values: "pwd", "git", "file". | Defaults to `git`.
```

Comment:
Rather than config, why not two methods, https://github.com/helix-editor/helix/pull/1600 and https://github.com/helix-editor/helix/pull/1352 ? Then the user can remap the default shortcut if they want

Reply:
That's better.